spec/bake_format_spec.rb

```diff
@@ -9,13 +9,20 @@
     expect(output.string).to eq("test {\n    test2\n}\n")
   end
 
-  it 'default indentation are two spaces' do
-    # todo: missing tests of root script
```

Comment:
with root script I mean bin/bake-format, your unittest "only" calls bake_format (format/bake_format). So most of the code lines are not covered, because the root script is not executed here.

Regarding Gemfile.lock: I don't use bundler, so this is your personal output from "bundle install". I will pull this change for you, but this will have no effect for my releases.